deployments/linea-goerli/usdc/migrations/1686918613_configurate_and_ens.ts

```diff
@@ -0,0 +1,240 @@
+const ENSTextRecordKey = 'v3-official-markets';
+const lineaCOMPAddress = '0xab3134fa5edfb3dc64aa790e8bb6448117d18fe9';
+
+export default migration('1686918613_configurate_and_ens', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+    const { utils } = ethers;
+
+    const {
+      bridgeReceiver,
+      comet,
+      cometAdmin,
+      configurator,
+      rewards
+    } = await deploymentManager.getContracts();
+
+    const {
+      lineaMessageService,
+      lineaL1TokenBridge,
+      governor,
+      COMP,
+      USDC
+    } = await govDeploymentManager.getContracts();
+
+    // ENS Setup
+    // See also: https://docs.ens.domains/contract-api-reference/name-processing
+    const ENSResolver = await govDeploymentManager.existing(
+      'ENSResolver',
+      ENSResolverAddress,
+      'goerli'
+    );
+    const subdomainHash = ethers.utils.namehash(ENSSubdomain);
+    const lineaGoerliChainId = (
+      await deploymentManager.hre.ethers.provider.getNetwork()
+    ).chainId.toString();
+    const newMarketObject = { baseSymbol: 'USDC', cometAddress: comet.address };
+    const officialMarketsJSON = JSON.parse(await ENSResolver.text(subdomainHash, ENSTextRecordKey));
+    if (officialMarketsJSON[lineaGoerliChainId]) {
+      officialMarketsJSON[lineaGoerliChainId].push(newMarketObject);
+    } else {
+      officialMarketsJSON[lineaGoerliChainId] = [newMarketObject];
+    }
+
+    const configuration = await getConfigurationStruct(deploymentManager);
+
+    const setConfigurationCalldata = await calldata(
+      configurator.populateTransaction.setConfiguration(comet.address, configuration)
+    );
+    const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, comet.address]
+    );
+    const setRewardConfigCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [comet.address, lineaCOMPAddress]
+    );
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address, cometAdmin.address, rewards.address],
+        [0, 0, 0],
+        [
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+          'deployAndUpgradeTo(address,address)',
+          'setRewardConfig(address,address)'
+        ],
+        [setConfigurationCalldata, deployAndUpgradeToCalldata, setRewardConfigCalldata]
+      ]
+    );
+
+    const COMPAmountToBridge = exp(1_000, 18);
+    const USDCAmountToBridge = exp(100_000, 18);
```

Comment:
This should be `exp(100_000, 6)` since USDC only uses 6 decimals.

Reply:
Right!